src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+</Callout>
+
+## Step 2: Configure the OpenTelemetry collector for New Relic [#configure]
+
+These examples are meant to serve as a starting point from which you can extend, customize, and validate configurations before using them in production.
+
+### Host monitoring using the host receiver [#host-receiver]
+
+The `collector-contrib` release provides a `hostreceiver` that generates metrics about the system scraped from various sources. It is intended to be used when the collector is deployed as an agent.
+
+When using the host receiver as part of the collector configuration, New Relic automatically detects host metrics as part of a `Host` entity and will synthesize its golden metrics providing the same experience as with the New Relic infrastructure agent. The following are the configuration requirements to enable the `Host` entity experience in New Relic UI:
+
+  * `host.id` attribute is present in host metrics.
+  * `service.name` and `container.id` attributes are not present in host metrics.
+
+Learn more about available metrics and advanced configurations from the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver).
+
+Adapt the `config.yaml` with these recommended parameters:
+
+<Callout variant="important">
+CPU, load, memory and disk utilization metrics require otelcol-contri release `v47` or greater.
+</Callout>
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "290px" }}>
+        Configuration
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        `receivers::hostmetrics`
+      </td>
+      <td>
+        Enable host metrics, 20 seconds interval is recommended (same default as the Infrastructure Agent), it should not be greater than 60 seconds to avoid issues with “host not responding” alerts. Process instrumentation is optional.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::resourcedetection`
+      </td>
+      <td>
+        Keep the following in mind:
+
+          * `env`: Reads resource information from the `OTEL_RESOURCE_ATTRIBUTES` environment variable.
+          * `system`: Adds `host.name` and `os.type`.
+          * For cloud environments, a specific resource detection processor needs to be configured so that metrics are decorated with `host.id` (required to identify the Host entity in New Relic One). Common cloud detectors are `gce` for GCP machines, `ec2` for AWS EC2 and `azure` for Azure VMs. Additional processors are available for orchestrated environments.
+          * For on-premises systems (or unsupported cloud environments), a `host.id` attribute is required. The `resource` processor can be used to copy the `host.name` value (from `system`) as a new `host.id` attribute. Note this value should be unique across all instrumented hosts:
```

Comment:
If you feel like revisiting passive voice throughout, this could be something like
Use the resource processor to copy...etc. Make sure this value is unique across all instrumented hosts.
I always worry when we use the word "should" does that mean, we don't require this, so you can do what you want at your peril? :-)

Reply:
@barbnewrelic, I fixed the two instance of passive voice in the table row. Thanks!